app/atoms/ln-patternlab/_ln-m-widget-contact.scss

```diff
@@ -0,0 +1,26 @@
+$_phone_prefix: 'T';
```

Comment:
These are just basic styles for the widgets and they will be customized on each project. I agree that we don't need the styles here, but I think is good to have them because it will be a way to show and standarize styling for ng-patternlab components. Also with these styles here, the widgets look better on the patterns page for ng-web-app as it comes.